packages/platform/sdk/src/lib/endpoints.ts

```diff
@@ -9,19 +9,19 @@ type BaseEndpointDeclaration = {
 // Define a type for static URIs
 type StaticUriEndpoint = BaseEndpointDeclaration & {
   uri: string;
-  constructUri?: never; // Ensure constructUri is not used here
+  constructUriFromObject?: never;
+  constructUriFromArray?: never;
 };
 
```

Comment:
Create a type for convenience to remove verbosity
`type StrArrOrRecordStrStr = string[] | Record<string, string>;`